packages/openapi/api/definitions/playlists.yaml

```
@@ -0,0 +1,574 @@
+title: playlists
+description: Definitions for playlists API
+resources:
+  playlists:
+    get:
+      operationId: playlists
+      tags:
+        - playlists
+      summary: Returns all playlists available in Sofie.
+      responses:
+        200:
+          description: Command successfully handled - returns an array of playlist Ids.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: number
+                    example: 200
+                  result:
+                    type: array
+                    items:
+                      type: string
+                    example: ['OKAgZmZ0Buc99lE_2uPPSKVbMrQ_']
+        500:
+          $ref: '#/components/responses/internalServerError'
+  activate:
+    put:
+      operationId: activate
+      tags:
+        - playlists
+      summary: Activates a Playlist.
+      parameters:
+        - name: playlistId
+          in: path
+          description: Playlist to activate.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Whether to activate into rehearsal mode.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                rehearsal:
+                  type: boolean
+      responses:
+        200:
+          $ref: '#/components/responses/putSuccess'
+        404:
+          $ref: '#/components/responses/playlistNotFound'
+        412:
+          description: There is already an active Playlist for the studio that the Playlist belongs to.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: number
+                    example: 412
+                  message:
+                    type: string
+                    example: Rundown Playlist is active, please deactivate before preparing it for broadcast
+        500:
+          $ref: '#/components/responses/internalServerError'
+  deactivate:
+    put:
+      operationId: deactivate
+      tags:
+        - playlists
+      summary: Deactivates a Playlist.
+      parameters:
+        - name: playlistId
+          in: path
+          description: Playlist to deactivate.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          $ref: '#/components/responses/putSuccess'
+        404:
+          $ref: '#/components/responses/playlistNotFound'
+        500:
+          $ref: '#/components/responses/internalServerError'
+  executeAdLib:
+    put:
```

Same as above: should this be a `post`? Executing adLibs is specifically not idempotent.
```suggestion
    post:
```